### ISSUE TITLE
emmet support

### ISSUE THREAD

**Issue by laggingreflex** [merwan7/sublime-marko#2]
[Emmet](http://emmet.io/) is a great library that allows you to use css-like shorthands for expanding into HTML tags. Its [sublime plugin](https://github.com/sergeche/emmet-sublime) works great with regular HTML files. But it doesn't work with marko files, more specifically when the file is set to marko syntax (using this plugin). 

I tried [this suggestion](http://stackoverflow.com/questions/16021147/sublime-text-emmet-tab-handler-in-other-files-than-html-without-ctrl-e) to put `"marko": {"extends": "html"}` in `Emmet/emmet/snippets.json` but it didn't work. 

It'd be great to have emmet support in marko. I'm not sure if this requires something to be done in this plugin, or the emmet-sublime plugin. If it is indeed with this plugin, please consider this feature request

**Comment by merwan7** [comment]
Hey @laggingreflex thanks for pointing this out.
I use Emmet as well and I love it, so it's important that this works.  Currently the way I use it is by typing `CTRL` + `E` when I want to expand something and that seems to work ok.  However, using tab like in a normal HTML file does not work.  This is due to Sublime-emmet's way of limiting itself to only certain file types to avoid messing w/ snippets.

I'm currently looking into this to see if I can add `Tab` support for it -- will update this issue when I have more info.

By the way, are you using ST2 or 3?

**Comment by laggingreflex** [comment]
I'm using ST3. I did not know about Ctrl+E. Tab would still be ideal though, but this essentially solves my issue. Thanks! 

I noticed the comment functionality (which is related to the other issue I opened up) is also provided by Emmet. But that doesn't work by its default shortcut (ctrl+/) either, however it does work from the command palette. I'm guessing this too has something to do with the emmet-sublime's limitation?
